src/app/util/settings-cookie.ts

```diff
@@ -0,0 +1,22 @@
+import Cookies from "js-cookie"
+
+import { type SettingsState } from "../slices"
+import packageJson from "../../../package.json"
+
+const COOKIE_NAME = packageJson.name
+
+export function getSettingFromCookies(): SettingsState | null {
+  const settingsCookie = Cookies.get(COOKIE_NAME)
+  if (settingsCookie) {
+    try {
+      return JSON.parse(settingsCookie) as SettingsState
+    } catch (e: any) {
+      console.error(`Error occurred while parsing settings from cookies: ${e}`)
+    }
+  }
+  return null
+}
```

Comment:
1. fix plural name - 1 cookie, many settings
1. simplify name
1. use `Cookies.get` default return type
1. return type is inferred
1. Use SERVICE_NAME instead (this is the same as packageJson.name)
```suggestion
export function getSettingsCookie() {
  const settingsCookie = Cookies.get(SERVICE_NAME)
  if (settingsCookie === undefined) return settingsCookie

  try {
    return JSON.parse(settingsCookie) as SettingsState
  } catch (e: any) {
    console.error(`Error occurred while parsing settings from cookies: ${e}`)
  }
}
```